packages/dds/merge-tree/src/partialLengths.ts

```diff
@@ -381,6 +381,8 @@ export class PartialSequenceLengths {
 	}
 
 	/**
+	 * PartialLengths for the leaves of a given MergeBlock.
```

Comment:
Nit: This isn't clear to me. Might be helpful to write this out as a complete sentence for clarity.